pkg/resource/acl/custom_set_output.go

```diff
@@ -0,0 +1,93 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package acl
+
+import (
+	"context"
+
+	svcsdk "github.com/aws/aws-sdk-go/service/memorydb"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	svcapitypes "github.com/aws-controllers-k8s/memorydb-controller/apis/v1alpha1"
+)
+
+const (
+	// The number of minutes worth of events to retrieve.
+	// 14 days in minutes
+	eventsDuration = 20160
+)
+
+func (rm *resourceManager) CustomDescribeACLsSetOutput(
```

Comment:
These functions shouldn't need to be made public. Instead they can be named `customDescribeACLsSetOutput`

Reply:
Yeah changed it to lower case c